src/Commands/CopyTemplatesCommand.php

```diff
@@ -52,46 +52,94 @@ public function copyTemplates(string $site_name) {
       touch($clone_dir . "/" . $directory . "/.gitkeep");
     }
     exec("git add -f db/.gitkeep logs/.gitkeep");
-    $iterator = new \DirectoryIterator($base_dir . '/templates');
-    for ($iterator->rewind(); $iterator->valid(); $iterator->next()) {
-      if (is_file($iterator->current()->getRealPath())) {
-        switch ($iterator->current()->getFilename()) {
-          case 'settings.local.php':
-            copy(
-              $iterator->current()->getRealPath(),
-              $clone_dir . '/web/sites/default/settings.local.php'
-            );
-            break;
-
-          case '.envrc':
-            $contents = file_get_contents($iterator->current()->getRealPath());
-            $contents = str_replace('**PROJECT_NAME**', $site_name, $contents);
-            $contents = str_replace('**PROJECT_PATH**', $clone_dir, $contents);
-            $contents = str_replace(
-              '**HASH_SALT**',
-              Crypt::randomBytesBase64(55),
-              $contents
-            );
-            file_put_contents($clone_dir . '/' . $iterator->current()
-                ->getFilename(), $contents);
-            break;
-
-          default:
-            copy(
-              $iterator->current()->getRealPath(),
-              $clone_dir . '/' . $iterator->current()->getFilename()
-            );
-        }
-      }
+
+    // We only have two types of templates -- drupal or wordpress. We might get any of a bunch of different types of frameworks back that we need to map to those two template types. Start by identifying some "known" allowed frameworks. This list may expand to include more frameworks.
+    $allowed_frameworks = [ 'wordpress', 'wordpress_network', 'drupal', 'drupal8' ];
+    // Identify the framework.
+    $framework = $this->getFramework( $site_name );
+    // If the framework isn't in the allowed_frameworks list, bail.
+    if ( ! in_array( $framework, $allowed_frameworks ) ) {
+      throw new \Exception( $framework );
     }
+    // Determine if the framework is WP or Drupal.
+    $framework = false === stripos( $framework, 'wordpress' ) ? 'drupal' : 'wordpress';
```

Comment:
I'm not sure if the logic that breaks the frameworks down into two paths should happen here or inside `copyFrameworkFiles`, tbh. It felt better to take this out of the `getFramework` command/function rather than what I originally commented about because that command can and should show the actual framework rather than just our interpretation, but the destination path is built in `copyFrameworkFiles` so I'm thinking this could maybe be moved to line 87-ish. Thoughts @stovak ?